spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/health/InMemorySpanCollectorMetricsHandler.java

```diff
@@ -0,0 +1,32 @@
+package org.springframework.cloud.sleuth.health;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * Implementation of the metrics statistics held in-memory.
+ *
+ * @author Marcin Grzejszczak
+ */
+public class InMemorySpanCollectorMetricsHandler implements SpanCollectorMetricsHandler {
```

Comment:
I don't think this should be the default, no-op should be the default. There's no benefit to incrementing atomic longs no-one will read.